framework/packages/abstract-client/Cargo.toml

```
@@ -11,6 +11,9 @@ keywords.workspace = true
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[features]
+test-utils = ["cw-asset", "abstract-testing"]
```

enable this by default? 

```suggestion
default = ["test-utils"]
test-utils = ["cw-asset", "abstract-testing"]
```